data/saas/config/rollbar_config.yml

```diff
@@ -0,0 +1,91 @@
+saas_config:
+  - fides_key: rollbar_connector
```

Comment:
The fides_key should be `<instance_fides_key>` after our SaaS connector template change. Refer to the other connectors for an example. Additionally, be sure to add this connector to `saas_connector_registry.toml`

Reply:
I'll update the dev guide with this!